R/dist.R

```diff
@@ -1372,3 +1372,48 @@ plot.dist_spec <- function(x, ...) {
     theme_bw()
   return(plot)
 }
+
+##' Fix the parameters of a `dist_spec`
+##'
+##' If the given `dist_spec` has any uncertainty, it is removed and the
+##' corresponding distribution converted into a fixed one.
+##' @return A `dist_spec` object without uncertainty
+##' @author Sebastian Funk
+##' @export
+##' @param x A [dist_spec] object
+##' @param strategy Character; either "center" (use the central estimate) or
```

Comment:
better to be specific and use mean here?

Reply:
not 100% convinced as I'm not sure whether "mean estimate of the mean" is really clearer than "central estimate of the mean" (and both are of course correct given that they're normal). I'd be inclined to leave it as is.

Reply:
that said "mean" is in line with the rest of the package terminology so probably better to stick with it